src/api/ledger/parse/trustline.js

```diff
@@ -7,16 +7,16 @@ const flags = utils.core.Transaction.flags.TrustSet;
 function parseTrustline(tx: Object): Object {
   assert(tx.TransactionType === 'TrustSet');
 
-  return {
+  return utils.removeUndefined({
     limit: tx.LimitAmount.value,
     currency: tx.LimitAmount.currency,
     counterparty: tx.LimitAmount.issuer,
     qualityIn: tx.QualityIn,
     qualityOut: tx.QualityOut,
-    disableRippling: (tx.Flags & flags.NoRipple) !== 0,
-    frozen: (tx.Flags & flags.SetFreeze) !== 0,
-    authorized: (tx.Flags & flags.SetAuth) !== 0
-  };
+    disableRippling: (tx.Flags & flags.NoRipple) !== 0 || undefined,
```

Comment:
```
  TrustSet: {
    SetAuth: 0x00010000,
    NoRipple: 0x00020000,
    SetNoRipple: 0x00020000,
    ClearNoRipple: 0x00040000,
    SetFreeze: 0x00100000,
    ClearFreeze: 0x00200000
  },
```
Some things to ponder:

There's clearly 3 states for a transaction's setting of freeze status: undefined, clearing and setting. Do we want the rule a bit more nuanced?  Also, compare frozen and authorized with disableRippling, where the latter is a verb.